regression/tests/studio/test_studio_textbooks.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test textbook page
+"""
+
+from bok_choy.web_app_test import WebAppTest
+
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.studio_textbooks import TextbookPageExtended
+from regression.tests.helpers import LoginHelper, get_course_info
+
+
+class TextbookTest(WebAppTest):
+    """
+    Test textbooks.
+    """
+    def setUp(self):
+        super(TextbookTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.textbook_page = TextbookPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'])
+
+        self.textbook_page.visit()
+
+    def test_textbook_edit_delete(self):
```

Comment:
you should break this out to be separate tests taht can be run independently. is that possible? ( I realize you're adjusting state... )